functionslab.js

```diff
@@ -0,0 +1,184 @@
+console.log('======question 1=======');
+
+function largTwoNum(num1, num2) {
+  if (num1 > num2){
+    return num1 + 'larger';
+  } else {
+    return num2
+  }
+
+}
+
+let result1 = largTwoNum(3, 2);
+console.log(result1);
+
+console.log('=========question 2===========');
+function getDogAge(num3, num4) {
+  return num3 + num3;
+}
+
+const getDogAgeES6 = (num3) => {
+  return num3 * 7;
+};
+
+let result2 = getDogAge(4);
+console.log(result2);
+console.log('========question3=========');
+
+let array2 = [13, 2, 4, -2];
+let result3 = 0;
+for (let i in array2) {
+  result3 += array2[i];
+}
+
+console.log(result3);
+console.log('=========question4===========');
+
+let sumReturn = (num4, num5) => {
+  return num4 + num5;
+};
+
+let result4 = sumReturn(14, 5);
+console.log(result4);
+console.log('========question5============');
+
+let numGrade = ['100', '99-90', '89-80', '79-70', '69-60'];
+let lettGrade = ['A', 'B', 'C', 'D', 'F'];
+for (let j = 0; j < numGrade.length; j++) {
+  console.log(lettGrade[j] + ',' + numGrade[j]);
+}
+
+const grCon = (grades) => {
```

Comment:
Logic is good but pay attention to indentations next time. It makes code look pretty and more readable when dealing with complicated logics with multiple statements.